autoload/ale/fix/registry.vim

```diff
@@ -285,6 +285,10 @@ let s:default_registry = {
 \       'suggested_filetypes': ['kt'],
 \       'description': 'Fix Kotlin files with ktlint.',
 \   },
+\   'styler': {
+\       'function': 'ale#fixers#styler#Fix',
+\       'suggested_filetypes': ['r'],
+\       'description': 'Fix R files with styler.',
 \   'latexindent': {
```

Comment:
Well I think you might be missing a `}` here?

Reply:
Right, thanks!

Reply:
Now tests pass, hooray 😄.

Reply:
Indeed :)